hash/saturn.xml

```diff
@@ -2535,12 +2552,30 @@ Olympic Soccer (Fra) T-7904H-09
 		<info name="alt_title" value="AI将棋"/>
 		<sharedfeat name="compatibility" value="NTSC-J"/>
 		<part name="cdrom" interface="cdrom">
+			<feature name="cd_matrix" value="T-18602GP-00413-P1K   V"/>
 			<diskarea name="cdrom">
 				<disk name="ai shougi (japan)" sha1="307542ffe38bc51517930eda0159d5a501c9807e" />
 			</diskarea>
 		</part>
 	</software>
 
+	<!-- CHD is missing index data from track 2; game uses them for audio playback. -->
+	<software name="aishogi2" supported="no">
+		<description>AI Shougi 2 (Japan)</description>
+		<year>1995</year>
+		<publisher>ASCII Something Good</publisher>
+		<info name="serial" value="T-17602G"/>
+		<info name="release" value="19980611"/>
+		<info name="alt_title" value="AI将棋 2"/>
+		<sharedfeat name="compatibility" value="NTSC-J"/>
+		<part name="cdrom" interface="cdrom">
+			<feature name="cd_matrix" value="T-17602GP-02272   A"/>
+			<diskarea name="cdrom">
+				<disk name="ai shougi 2 (japan)" sha1="6c42d91ce7ace93baeccabf4aac80c39a93c4d21" />
+			</diskarea>
+		</part>
+	</software>
```

Comment:
We really need “bad dump” flags on things missing important data.

Reply:
Gotcha, I'll add bad dump to the items in this PR later today.

I only realized this after searching for info on that chdman issue I posted the other day. It seems all of the discs listed in this comment have the issue: https://github.com/mamedev/mame/issues/10308#issuecomment-2670620037

Do you want me to add the baddump flag and notes to all of these, or do I wait until I get to checking and resubmitting them? I'm gonna take a while to get to all of them, so I can do it already if that's okay with you.